### ISSUE TITLE
VisJS has disappeared

### ISSUE THREAD

**Issue by Zabeus** [visjs-community/vis#1]
Not sure the appropriate place to report this, assuming you don't already know. The visjs website returns 404, and https://github.com/almende/ have removed Vis from their list of projects.

**Comment by tmkasun** [comment]
Yes, What happened to repository , I'm too getting 404 and can't find the repository ?

**Comment by tmkasun** [comment]
Found https://github.com/almende/vis/issues/4259 issue from google cache and it seems they have moved it here.

**Comment by Zabeus** [comment]
I guess the website just needs to be redirected?

The example pages are all in the repo.

2019年5月25日(土) 7:47 Kasun Thennakoon <notifications@github.com>:

**Comment by Jogai** [comment]
Ok, I completely missed this, but apperantly the development now happens here: https://github.com/visjs/vis-timeline
